src/components/Table.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <n-space vertical :size="12">
+    <n-data-table
+      ref="table"
+      :columns="columns"
+      :data="tableData"
+      :pagination="pagination"
+      :scroll-x="scrollLength"
+    />
+  </n-space>
+</template>
+
+<script>
+import { NSpace, NDataTable } from "naive-ui";
+
+const keys = {};
+
+keys.total = ["cases", "deaths", "active", "critical", "admission", "recovered", "tests"];
+keys.daily = ["todayCases", "todayDeaths", "todayAdmission", "todayRecovered"];
+keys.rate = keys.total.map((i) => i + "PerOneMillion");
```

Comment:
```suggestion
const totalKeys = ["cases", "deaths", "active", "critical", "admission", "recovered", "tests"];
const dailyKeys = ["todayCases", "todayDeaths", "todayAdmission", "todayRecovered"];
const rateKeys = totalKeys.map(key => `${key}PerOneMillion`;

const keys = {
  total: totalKeys,
  daily: dailyKeys,
  rate: rateKeys,
};
```

and now TypeScript knows what `keys` is

Reply:
Interesting